controllers/tmf-apis/catalog.js

```diff
@@ -269,131 +269,143 @@ var catalog = (function() {
             }
         }
 
-        // Check the offering categories
-        var categories = newBody ? newBody.category : [];
-
-        async.eachSeries(categories, function (category, taskCallback) {
-
-            var categoryApiUrl = url.parse(category.href).pathname;
+        async.series([
+            function (callback) {
+                // Check the RS model
+                if (newBody && newBody.serviceCandidate) {
```

Comment:
Doing the validation in this way, it is possible to create an offering without RS model, take into account that if the operation is a creation and the provider do not include a serviceCandidate field you are not going to call the method.